src/garage/sampler/is_sampler.py

```diff
@@ -1,23 +1,25 @@
+"""ISSampler."""
 import copy
 from math import exp
 from math import log
 import random
 
 from numpy import var
 
-from garage.sampler import BatchSampler
+from garage.sampler import Sampler
 from garage.sampler.utils import truncate_paths
 
 
-class ISSampler(BatchSampler):
-    """
+class ISSampler(Sampler):
+    """ISSampler.
```

Comment:
Please don't use the class name as the first line of the comment. I'm surprised this passes the linter.

See http://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings for a guide on how to write good docstrings.

Reply:
This should be `Importance sampling sampler`.